years/2018/Students/vvalada3.txt

```diff
@@ -0,0 +1,36 @@
+                  ####
+            #    #    ########
+           #  ### ####XXXXXXXX######    ####
+            X ## # XXXXXXXXXXXXXXXXX##########
+         ## ##   #X       XXXXXXXXXXXXX#XXXX####
+       ##  X#   ##XXXXXXXX      XXXXXXXXXXXXXX#X#
+     ##  XXXX#####XXXXXXXXXXXXXX              X#X##
+    #  XXXXXXX##XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX   XX#
+  #  XXXXXXXXX#XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX  X##
+ #X##XXXXXXXXX####XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX XX#
+ # XXXXXXXXXXXXXXX########XXXXXXXXXXXXXXXXXXXXXXXXXX  X#
+ # XXXXXXXXX######XXXXXXXX#XXXXXXXXXXXXXXXXXXXXXXXXXXX X##   ########
+  ##########X X X XXXXXXXX#XXXXXXXXXXXXXXXXXXXXXXXXXXXX XX###XXXXXXXX#
+   X X X X X X X X X X #####XXXXXXXXXXXXXXXXXXXXXXXXXXXX  XX##XXXXXXXX#
+     #X X X X X XXXX#XXXXXXX####XXXXXXXXXXXXXXXXXXXXXXXXXX XXX##XXXXXX#
+       X X X X X XXXXX X X XXXXX##XXXXXXXXXXXXXXXXXXXXXXXXX  XXX#XXXX#
+        X X X X XX#X#XXXXXXXXXXXXX#XXXXXXXXXXXXXXXXXX########X########
+              #X X XXX X X X X XXX#XXXXXXXXXXXXXX###XXXXXXXXX#X#XXXXX#
+               #XXX##X#XXXXXXXXXXX#XXXXXXXXXXX###X#X#X#XXXXXXX#X#XXXX#
+               # X X##XXXXXX X X XX#XXXXXXXX##X##XX#X#X#XXXXXXX#XXX#XX#
+               ####X#X#####XXXXXXXX#XXXXXX##X##XXXXXXX#X#X#XXX#XXX#XX##
+              #X # # X X #XX X XXX#XXXX###XX#XXXXXXXX#X#X#XXX#XXX####
+             ##XXX### X X#XXXXXX##XXX##XXX#XXXXXXXXX#X#X#XXXXXX###
+          ###X X ### # X#X X XX#XXX##XXXX#X#XXXXXXXXX#X#XXXXX####
+       ###XXXXXX########XXXXXX# XXX#XXX##XXX#XXXXXXX#X#XXX##XX#X#
+    ############       X X #X# X X #X##XXXX#XXXXXXX#X#X#X#X#XX#X
+      ###     ##      #XXX#X# X X #XXXXXXX#X#XXXXX#X###XXX#XX#X
+     # #     ##      # X #X# X X X#XXXXXX#X#XXXXXXX#XXXXXXX##X
+      #            ##XXX#X# X X X #XXXXX#X#XXXXX###X#XXXXX##X
+             ######X X #X#   X X X#XXXX#X#X#X###XXX#X##### X
```

Comment:
This will get cut off here

Reply:
@val500 the max height for the ascii art is 30 rows and the max width is 80 characters